skrub/_joiner.py

```diff
@@ -271,7 +261,7 @@ def _check_max_dist(self):
     def _check_ref_dist(self):
         if self.ref_dist not in _MATCHERS:
             raise ValueError(
-                f"ref_dist should be one of {list(_MATCHERS.keys())}, got"
+                f"`ref_dist` should be one of {list(_MATCHERS.keys())}, got"
```

Comment:
this doesn't get rendered so we don't really need markub maybe we could use regular quotes instead